torch/testing/_internal/common_quantization.py

```diff
@@ -130,16 +130,27 @@ def wrapper(*args, **kwargs):
             fn(*args, **kwargs)
     return wrapper
 
+def get_script_module(model, tracing, data):
```

Comment:
is this used?

Reply:
this is used in test, you probably missed the changes in `test_quantize_script.py`, it's too large so the change is not displayed by default in github..